DependencyInjection/SonataDoctrinePHPCRAdminExtension.php

```diff
@@ -83,79 +88,56 @@ public function getNamespace()
     }
 
     /**
-     * Set the tree type mapping configuration in the services.
-     *
-     * @param array            $config
-     * @param ContainerBuilder $container
+     * {@inheritdoc}
      */
-    private function loadTreeTypes($config, ContainerBuilder $container)
+    public function prepend(ContainerBuilder $container)
```

Comment:
so to prepend configuration, we load our own configuration already and process it, then process the cmf_resource configuration and prepend values from that to our own configuration? this feels super complicated. what exactly are we trying to achieve with this? i think we need to find a more straightforward solution. maybe i have an idea when i understand what exactly we try to do here.